app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -40,10 +40,59 @@ Vue.mixin({
         vm.staffOptions();
         vm.fetchIfPendingFulfillments();
         vm.initChain();
+        vm.eventParams();
       }).catch(function(error) {
         vm.loadingState = 'error';
         _alert('Error fetching bounties. Please contact founders@gitcoin.co', 'error');
       });
+    },
+    eventParams: function() {
+      let searchParams = new URLSearchParams(window.location.search);
+
+      if (searchParams.has('mutate_worker_action')) {
+        let action = searchParams.get('mutate_worker_action');
+        let product = document.result.event ? 'hackathons' : 'bounties';
+        let productPersona = document.result.event ? 'hackathon' : 'bounty';
+        let persona = this.isOwner ? 'funder' : 'hunter';
+        let mtcPersona = `${productPersona}-${persona}`;
+        let bountyEvent = {
+          'alias': 'products',
+          'data': [
+            {
+              'name': 'product',
+              'attributes': {
+                'product': product,
+                'persona': mtcPersona,
+                'action': action
+              }
+            }
+          ]
+        };
+
+        if (document.result.event) {
+          let stopHackathonEvent = {
+            'alias': 'hackathon',
+            'data': [
+              {
+                'name': action,
+                'attributes': {
+                  'hackathon-slug': document.result.event.slug,
```

Comment:
Worth adding null check?

Reply:
So slug cannot be null for hackathons but Im already setting this inside if document.result.event so only will be trigger if a hackathon